sdk/servicebus/Azure.Messaging.ServiceBus/README.md

```diff
@@ -336,7 +336,7 @@ try
     // Receive messages using the receiver client
 }
 catch (ServiceBusExceptions ex) where 
-    (ex.Reason == ServiceBusException.FailureReason.ServiceTimeout)
+    (ex.Reason == ServiceBusFailureReason.ServiceTimeout)
```

Comment:
The above has a typo.    `where` should be `when`.    _(that's on me.  You inherited that.  :smile: )_

Reply:
Fixed! ☺